stp_zmq/zstack.py

```diff
@@ -717,7 +717,9 @@ def sendPingPong(self, remote: Union[str, Remote], is_ping=True):
         msg = self.pingMessage if is_ping else self.pongMessage
         action = 'ping' if is_ping else 'pong'
         name = remote if isinstance(remote, (str, bytes)) else remote.name
-        r = self.send(msg, name)
+        # Do not use Batches for sending health messages
+        r = self.transmit(msg, name, is_batch=False)
```

Comment:
Are we still need in ```is_batch``` for ```transmit()```?
 ```is_batch``` is used in ```send()``` to call ```transmit()```. But it doesn't use this parm.

Reply:
We currently don't use it anymore. I decided not to remove it for now in case we will need this information again.